README.md

```diff
@@ -142,6 +143,13 @@ Browserslist will take queries from tool option,
   than Microsoft Edge. Chinese QQ Browsers has more market share than Firefox
   and desktop Safari combined.
 
+## Why you need to call update-db script regularly?
```

Comment:
We are not mentioning `last 2 versions` case, that updating version will keep it up to date